### ISSUE TITLE
related to ydata-sdk Synthesizer model

### ISSUE THREAD

**Issue by range-yugen** [ydataai/ydata-synthetic#291]
**Describe the bug**
we are using the y_data synthesiser for almost 2 months to generate synthetic data. but for the last 3 days, it is not able to train "ydata-sdk Synthesizer" model. we try to train other models like gan, ctgan etc. in it which are training perfectly and can generate synthetic data from those trained models but this is not possible with ydata-sdk Synthesizer model.

**To Reproduce**
1. run stramlit app
2. click on Train a synthesizer 
4. select csv file and numericals and categorical columns
5. select your model = ydata-sdk Synthesizer
6. provide SDK Token from YData account (showing valid, green tick)
7. click on "click here to start the training process"
8. see the error

**Expected behavior**
earlier there was no such error when using ydata-sdk Synthesizer model. and able to generate synthetic data easily.

**Screenshots**
![image](https://github.com/ydataai/ydata-synthetic/assets/105635408/82a2db5d-3920-4068-8ca1-ff8f1c2b866d)

**Desktop (please complete the following information):**
 - OS: [window]
 - Browser [brave, edge]
 - Version [window 11]
 - Cuda version

**Additional context**
library version, which i am using
Package                      Version
---------------------------- -----------
absl-py                      1.4.0
aiofiles                     22.1.0
aiosqlite                    0.18.0
altair                       4.1.0
anyio                        3.5.0
argon2-cffi                  21.3.0
argon2-cffi-bindings         21.2.0
asttokens                    2.0.5
astunparse                   1.6.3
atomicwrites                 1.4.1
attrs                        22.1.0
Babel                        2.11.0
backcall                     0.2.0
beautifulsoup4               4.12.2
bleach                       4.1.0
blinker                      1.4
Bottleneck                   1.3.5
brotlipy                     0.7.0
cachetools                   4.2.2
certifi                      2023.5.7
cffi                         1.15.1
charset-normalizer           2.0.4
click                        8.0.4
cloudpickle                  2.2.1
colorama                     0.4.6
comm                         0.1.2
contourpy                    1.1.0
cryptography                 39.0.1
ctgan                        0.7.3
cycler                       0.11.0
dacite                       1.8.1
data                         0.4
debugpy                      1.5.1
decorator                    5.1.1
defusedxml                   0.7.1
dm-tree                      0.1.8
dython                       0.5.1
easydict                     1.10
entrypoints                  0.4
exceptiongroup               1.1.2
executing                    0.8.3
Faker                        18.11.2
fastjsonschema               2.16.2
filelock                     3.12.2
flatbuffers                  23.5.26
fonttools                    4.40.0
funcsigs                     1.0.2
gast                         0.4.0
gitdb                        4.0.7
GitPython                    3.1.30
google-auth                  2.21.0
google-auth-oauthlib         0.4.6
google-pasta                 0.2.0
grpcio                       1.56.0
h11                          0.14.0
h5py                         3.9.0
htmlmin                      0.1.12
httpcore                     0.16.3
httpx                        0.23.3
idna                         3.4
ImageHash                    4.3.1
importlib-metadata           6.0.0
iniconfig                    2.0.0
ipykernel                    6.19.2
ipython                      8.12.0
ipython-genutils             0.2.0
ipywidgets                   8.0.4
jedi                         0.18.1
Jinja2                       3.1.2
joblib                       1.3.1
json5                        0.9.6
jsonschema                   4.17.3
jupyter                      1.0.0
jupyter_client               8.1.0
jupyter-console              6.6.3
jupyter_core                 5.3.0
jupyter-events               0.6.3
jupyter_server               2.5.0
jupyter_server_fileid        0.9.0
jupyter_server_terminals     0.4.4
jupyter_server_ydoc          0.8.0
jupyter-ydoc                 0.2.4
jupyterlab                   3.6.3
jupyterlab-pygments          0.1.2
jupyterlab_server            2.22.0
jupyterlab-widgets           3.0.5
keras                        2.11.0
kiwisolver                   1.4.4
libclang                     16.0.0
lxml                         4.9.2
Markdown                     3.4.3
markdown-it-py               2.2.0
MarkupSafe                   2.1.1
matplotlib                   3.6.3
matplotlib-inline            0.1.6
mdurl                        0.1.0
mistune                      0.8.4
mkl-fft                      1.3.6
mkl-random                   1.2.2
mkl-service                  2.4.0
mpmath                       1.3.0
multimethod                  1.9.1
nb-conda-kernels             2.3.1
nbclassic                    0.5.5
nbclient                     0.5.13
nbconvert                    6.5.4
nbformat                     5.7.0
nest-asyncio                 1.5.6
networkx                     3.1
notebook                     6.5.4
notebook_shim                0.2.2
numexpr                      2.8.4
numpy                        1.23.5
oauthlib                     3.2.2
opt-einsum                   3.3.0
packaging                    21.3
pandas                       2.0.3
pandas-profiling             3.6.6
pandocfilters                1.5.0
parso                        0.8.3
patsy                        0.5.3
phik                         0.12.3
pickleshare                  0.7.5
Pillow                       9.4.0
pip                          23.1.2
platformdirs                 2.5.2
pluggy                       1.2.0
ply                          3.11
pmlb                         1.0.1.post3
prettytable                  3.6.0
prometheus-client            0.14.1
prompt-toolkit               3.0.36
protobuf                     3.19.6
psutil                       5.9.0
pure-eval                    0.2.2
py                           1.11.0
pyarrow                      11.0.0
pyasn1                       0.5.0
pyasn1-modules               0.3.0
pycparser                    2.21
pydantic                     1.10.9
pydeck                       0.7.1
Pygments                     2.15.1
Pympler                      0.9
pyOpenSSL                    23.0.0
pyparsing                    3.1.0
PyQt5                        5.15.7
PyQt5-sip                    12.11.0
pyrsistent                   0.18.0
PySocks                      1.7.1
pytest                       6.2.5
python-dateutil              2.8.2
python-json-logger           2.0.7
pytz                         2022.7
PyWavelets                   1.4.1
pywin32                      305.1
pywinpty                     2.0.10
PyYAML                       6.0
pyzmq                        25.1.0
qtconsole                    5.4.2
QtPy                         2.2.0
rdt                          1.5.0
requests                     2.30.0
requests-oauthlib            1.3.1
rfc3339-validator            0.1.4
rfc3986                      1.5.0
rfc3986-validator            0.1.1
rich                         13.3.5
rsa                          4.9
scikit-learn                 1.2.2
scipy                        1.10.1
seaborn                      0.11.1
semver                       2.13.0
Send2Trash                   1.8.0
setuptools                   67.8.0
sip                          6.6.2
six                          1.16.0
smmap                        4.0.0
sniffio                      1.2.0
soupsieve                    2.4
stack-data                   0.2.0
statsmodels                  0.14.0
streamlit                    1.16.0
streamlit-pandas-profiling   0.1.3
sympy                        1.12
table-evaluator              1.4.2
tangled-up-in-unicode        0.2.0
tensorboard                  2.11.2
tensorboard-data-server      0.6.1
tensorboard-plugin-wit       1.8.1
tensorflow                   2.11.1
tensorflow-estimator         2.11.0
tensorflow-intel             2.11.1
tensorflow-io-gcs-filesystem 0.31.0
tensorflow-probability       0.19.0
termcolor                    2.3.0
terminado                    0.17.1
threadpoolctl                3.1.0
tinycss2                     1.2.1
toml                         0.10.2
tomli                        2.0.1
toolz                        0.12.0
torch                        2.0.1
tornado                      6.2
tqdm                         4.65.0
traitlets                    5.7.1
typeguard                    2.13.3
typing_extensions            4.6.3
tzdata                       2023.3
tzlocal                      2.1
urllib3                      1.26.16
validators                   0.18.2
visions                      0.7.5
watchdog                     2.1.6
wcwidth                      0.2.5
webencodings                 0.5.1
websocket-client             0.58.0
Werkzeug                     2.3.6
wheel                        0.38.4
widgetsnbextension           4.0.5
win-inet-pton                1.1.0
wordcloud                    1.9.2
wrapt                        1.15.0
y-py                         0.5.9
ydata                        0.2
ydata-core                   0.3.0
ydata-datascience            0.3.0
ydata-profiling              4.3.1
ydata-sdk                    0.6.0
ydata-synthetic              1.2.0
ypy-websocket                0.8.2
zipp                         3.11.0

**Comment by heedoooch** [comment]
I am facing the same issue!

**Comment by gmartinsribeiro** [comment]
Thanks for bringing this to our attention.
While we troubleshoot the issue please use [Fabric](https://fabric.ydata.ai/) directly - easier GUI for you to use the best performing models.
